src/components/UserBox.vue

```diff
@@ -402,8 +413,10 @@ export default {
     padding: 1em;
     text-align: center;
     margin: 0;
+    user-select: none;
     box-sizing: border-box;
 
+    .kiwi-userbox-plugin-actions a,
```

Comment:
What is this selector referring to?

Reply:
This is styled because plugins may add simple links in, instead of always using 'kiwi-userbox-plugin-action' class :)